core.coffee

```diff
@@ -407,6 +407,19 @@
 			delete: (params) ->
 				return @request(params, method: 'DELETE')
 
+			batch: (data) ->
+				params =
+					method: 'POST'
+					url: '$batch'
+				overrides =
```

Comment:
This is a bit iffy as not every backed supports `multipart/mixed` requests. For example in nodejs superagent does not support that kind of request and I am not sure what the situation for the browser world. 
We could attempt to shim it if it is not present on the backend... this should not be too hard since we are already making use of `mimemessage` for the changesets (they have to be formatted as `multipart/mixed` requests). In that case we would rely on some `raw` method that allows us to build the raw http request.